cryton/worker/empire.py

```diff
@@ -240,7 +240,7 @@ async def deploy_agent(arguments: dict) -> dict:
         logger.logger.debug(
             "Deploying agent via MSF session.", session_id=session_id, payload=payload, target_ip=target_ip
         )
-        session_to_use.execute_in_shell(payload)
+        session_to_use.execute_in_shell(executable=''.join(payload.split()[:1]).strip(), arguments=payload.split()[1:])
```

Comment:
Could I suggest a temporary variable?
```suggestion
        payload_tmp = payload.split()
        session_to_use.execute_in_shell(payload_tmp[0], payload_tmp[1:])
```

Reply:
Addressed in [dc336a6](https://github.com/CSIRT-MU/Cryton/pull/1/commits/dc336a6e825faeb503e169d4885a43a21ab3d08f), thanks for the input!